cluster/centos/util.sh

```diff
@@ -316,10 +333,34 @@ function provision-node() {
 #   KUBE_TEMP
 function ensure-setup-dir() {
   kube-ssh "${1}" "mkdir -p ${KUBE_TEMP}; \
+                   mkdir -p ${KUBE_TEMP}/etcd-cert; \
                    sudo mkdir -p /opt/kubernetes/bin; \
                    sudo mkdir -p /opt/kubernetes/cfg"
 }
 
+# Generate certificates for etcd cluster
+#
+# Assumed vars:
+#   $1 (etcd member name)
+#   $2 (master ip)
+function ensure-etcd-cert() {
+  local etcd_name="$1"
+  local master_ip="$2"
+  local cert_dir="${ROOT}/etcd-cert"
+
+  mkdir -p "${cert_dir}"
+  pushd "${cert_dir}"
```

Comment:
Actually, I need to check `client.pem` or `client-key.pem` exists in the `${cert_dir}` directory. Use `generate-etcd-cert "${cert_dir}"` instead of `generate-etcd-cert .` seem could be better.